<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>CheckItem</class>
	<widget class="QFrame" name="CheckItem">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>400</width>
				<height>60</height>
			</rect>
		</property>
		<property name="frameShape">
			<enum>QFrame::Panel</enum>
		</property>
		<property name="frameShadow">
			<enum>QFrame::Raised</enum>
		</property>
		<property name="lineWidth">
			<number>1</number>
		</property>
		<property name="midLineWidth">
			<number>2</number>
		</property>
		<property name="styleSheet">
			<string>QFrame#CheckItem { background-color: rgb(100, 100, 100); }</string>
		</property>
		<property name="windowTitle">
			<string>Check Item</string>
		</property>
		<property name="sizePolicy">
			<sizepolicy hsizetype="Preferred" vsizetype="Fixed">
				<horstretch>0</horstretch>
				<verstretch>0</verstretch>
			</sizepolicy>
		</property>
		<property name="minimumSize">
			<size>
				<width>0</width>
				<height>60</height>
			</size>
		</property>
		<property name="maximumSize">
			<size>
				<width>16777215</width>
				<height>60</height>
			</size>
		</property>
		<layout class="QHBoxLayout" name="main_layout" stretch="2,2,3">
			<property name="spacing">
				<number>10</number>
			</property>
			<property name="leftMargin">
				<number>10</number>
			</property>
			<property name="topMargin">
				<number>1</number>
			</property>
			<property name="rightMargin">
				<number>10</number>
			</property>
			<property name="bottomMargin">
				<number>1</number>
			</property>
			<item>
				<widget class="QLabel" name="item_label">
					<property name="text">
						<string>체크 항목</string>
					</property>
					<property name="font">
						<font>
							<pointsize>12</pointsize>
						</font>
					</property>
				</widget>
			</item>
			<item>
				<layout class="QHBoxLayout" name="radio_layout">
					<property name="spacing">
						<number>5</number>
					</property>
					<property name="sizeConstraint">
						<enum>QLayout::SetDefaultConstraint</enum>
					</property>
					<item>
						<spacer name="radio_left_spacer">
							<property name="orientation">
								<enum>Qt::Horizontal</enum>
							</property>
							<property name="sizeHint" stdset="0">
								<size>
									<width>40</width>
									<height>20</height>
								</size>
							</property>
						</spacer>
					</item>
					<item>
						<widget class="QRadioButton" name="pass_radio">
							<property name="text">
								<string>Pass</string>
							</property>
							<property name="font">
								<font>
									<pointsize>12</pointsize>
								</font>
							</property>
						</widget>
					</item>
					<item>
						<widget class="QRadioButton" name="fail_radio">
							<property name="text">
								<string>Fail</string>
							</property>
							<property name="font">
								<font>
									<pointsize>12</pointsize>
								</font>
							</property>
						</widget>
					</item>
					<item>
						<widget class="QRadioButton" name="na_radio">
							<property name="text">
								<string>N/A</string>
							</property>
							<property name="font">
								<font>
									<pointsize>12</pointsize>
								</font>
							</property>
						</widget>
					</item>
				</layout>
			</item>
			<item>
				<widget class="QTextEdit" name="comment_text">
					<property name="font">
						<font>
							<pointsize>10</pointsize>
						</font>
					</property>
					<property name="minimumSize">
						<size>
							<width>0</width>
							<height>40</height>
						</size>
					</property>
					<property name="maximumSize">
						<size>
							<width>16777215</width>
							<height>40</height>
						</size>
					</property>
					<property name="placeholderText">
						<string>비고사항을 입력하세요...</string>
					</property>
				</widget>
			</item>

		</layout>
	</widget>
	<resources />
	<connections />
</ui>